<?xml version="1.0"?>
<launch>
    <arg name="action_dispatch_topic" default="/kcl_rosplan/action_dispatch"/>
    <arg name="action_feedback_topic" default="/kcl_rosplan/action_feedback"/>

    <node pkg="mdr_move_base_action" type="move_base_client" name="mdr_move_base_client" output="screen">
        <param name="action_name" type="str" value="move_base" />
        <param name="server_name" type="str" value="move_base_server" />
        <param name="action_timeout" type="double" value="120" />
        <remap from="action_dispatch_topic" to="$(arg action_dispatch_topic)" />
        <remap from="action_feedback_topic" to="$(arg action_feedback_topic)" />
    </node>
</launch>
